client/web/src/search/home/SearchPageInput.tsx

```diff
@@ -83,10 +83,18 @@ export const SearchPageInput: React.FunctionComponent<React.PropsWithChildren<Pr
                       searchQueryHistorySource({
                           userId: props.authenticatedUser.id,
                           selectedSearchContext: props.selectedSearchContextSpec,
+                          onSelection: index => {
+                              props.telemetryService.log('SearchHistoryAutcompleteSuggestionClicked', { index })
```

Comment:
```suggestion
                              props.telemetryService.log('SearchHistoryAutocompleteItemClicked', { index })
```

Fixed typo, and also thinking "Item" is simpler. Don't think this has any side effects?